blivet/populator/helpers/__init__.py

```diff
@@ -1,4 +1,5 @@
 import inspect as _inspect
+import six as _six
```

Comment:
The goal was to keep the `blivet.populator.helpers` package namespace uncluttered.